www/assessment/customization/style-guide/.eslintrc

```diff
@@ -0,0 +1,78 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "airbnb",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "ignorePatterns": [],
+  "plugins": [
+    "@typescript-eslint",
+    "react-hooks"
+  ],
+  "globals": {
+    "__LRN_DEVELOPMENT": true,
```

Comment:
This is not necessary as you are not building a proper client facing API that needs to differentiate bw DEV vs PROD